ykman/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: Daniel Everett <deverett@gmail.com>
+# Maintainer: Daniel Everett <deverett@gmail.com>
+pkgname=yubikey-manager
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Python library and command line tool for configuring any YubiKey over all USB interfaces"
+url="https://github.com/dae3/alpine-yubikey-manager"
+arch="noarch"
+license="BSD-2-Clause"
+depends="python3 py-setuptools libusb yubikey-personalization-dev libyubikey pcsc-lite pcsc-lite-libs"
+depends_dev="py3-pip"
+makedepends="$depends_dev"
+install=""
+subpackages=""
+source="https://github.com/Yubico/yubikey-manager/releases/download/yubikey-manager-0.7.1/yubikey-manager-0.7.1.tar.gz"
```

Comment:
Should be:
https://github.com/Yubico/$pkgname/releases/download/$pkgname-$pkgver/$pkgname-$pkgver.tar.gz
so we don't change the source url on every upgrade.